firebase-storage/test-app/src/main/java/com/example/storage/MainActivity.java

```diff
@@ -104,6 +105,7 @@ protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.activity_main);
 
+    FirebaseApp.initializeApp(getApplicationContext());
```

Comment:
OOC, why this line weren't there before.

Reply:
This lets us use the test app without the Google Services plugin, which we had to remove as it breaks the CI build.